.github/workflows/sync-upstream.yaml

```diff
@@ -0,0 +1,28 @@
+name: sync-upstream
```

Comment:
Please confirm your diff.

Reply:
Reverted unnecessary fixes.

Reply:
Mm, there is no change. :pleading_face:

Reply:
Please remove `sync-upstream.yaml`.

Reply:
Also, please confirm the pull request guidelines.